src/instructlab/lab.py

```diff
@@ -1090,37 +1090,20 @@ def train(
 
         # TODO: Figure out what to do when there are multiple checkpoint dirs.
         # Right now it's just copying files from the first one numerically not necessarily the best one
-        added_tokens_file = glob(
-            training_results_dir + "/checkpoint-*/added_tokens.json"
-        )
-        special_tokens_map = glob(
-            training_results_dir + "/checkpoint-*/special_tokens_map.json"
-        )
-        tokenizer_json = glob(training_results_dir + "/checkpoint-*/tokenizer.json")
-        tokenizer_model = glob(training_results_dir + "/checkpoint-*/tokenizer.model")
-        tokenizer_config_json = glob(
-            training_results_dir + "/checkpoint-*/tokenizer_config.json"
-        )
-        config_json = glob(training_results_dir + "/merged_model/config.json")
-        generation_config_json = glob(
-            training_results_dir + "/merged_model/generation_config.json"
-        )
-        safe_tensors = glob(training_results_dir + "/merged_model/*.safetensors")
+        for fpath in (
+            "/checkpoint-*/added_tokens.json",
+            "/checkpoint-*/special_tokens_map.json",
+            "/checkpoint-*/tokenizer.json",
```

Comment:
Would it make sense to do `"/checkpoint-*/tokenizer.*",` to further refactor?

Reply:
This is marked as outdated since I've put the review in an early commit.`

Reply:
I dunno. Are these the **only** three files in the directory?